InternetShop/InternetShop/Controllers/ProductController.cs

```diff
@@ -29,13 +29,21 @@ public async Task<IEnumerable<ProductViewModel>> GetAll(CancellationToken cancel
         }
 
         [HttpGet("{id}")]
-        public async Task<ProductViewModel?> GetById(int id, CancellationToken cancellationToken)
+        public async Task<ProductViewModel?> GetById([FromRoute] int id, CancellationToken cancellationToken)
         {
             var product = await _productService.GetById(id, cancellationToken);
             var mappedProduct = _mapper.Map<ProductModel>(product);
             return _mapper.Map<ProductViewModel>(mappedProduct);
         }
 
+        [HttpGet("search")]
```

Comment:
incorret name, because idon't understand what this method do. F.e GetProductByPrice or something like that